test/tools/libtesteth/Options.cpp

```diff
@@ -72,6 +74,27 @@ void printVersion()
 {
     cout << prepareVersionString() << "\n";
 }
+}
+
+void Options::setVerbosity(int _level)
+{
+    static std::ostringstream strCout;  // static string to redirect logs to
+    static dev::LoggingOptions logOptions;
+    if (_level > verbosity)
+    {
+        verbosity = _level;
+        logOptions.verbosity = verbosity;
+        if (_level <= 0)
+        {
+            logOptions.verbosity = -1;
+            std::cout.rdbuf(strCout.rdbuf());
+            std::cerr.rdbuf(strCout.rdbuf());
+        }
+        else if (_level == 1 || _level == 2)
+            logOptions.verbosity = -1;
+        dev::setupLogging(logOptions);
```

Comment:
the message is not being output twice. 
if you want to call setVerbosity once. then there is no need in setVerbosity. 
I just copy if (verbosity > x)  to all places where it was called and then call setupLogging

Reply:
Yeah right, that's what I meant, I think it would be simpler (I'd still keep `setVerbosity` for the sake of smaller functions)
But ok, it's up to you,